dist/focus-ring.js

```diff
@@ -265,7 +225,6 @@ document.addEventListener('DOMContentLoaded', function() {
     if (index(el).contains('focus-ring'))
       return;
     index(el).add('focus-ring');
-    el.setAttribute('data-focus-ring-added', '');
```

Comment:
@alice [bug is filed](https://github.com/WICG/focus-ring/issues/35). And this PR has been updated to restore `data-focus-ring-added`.